core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/reservoirsampling/ReservoirSamplingOpDesc.scala

```diff
@@ -28,18 +29,11 @@ class ReservoirSamplingOpDesc extends OperatorDescriptor {
   // In order to make sure the total output is k, each executor should produce (k / n) items
   // (n is the number of the executors)
   @JsonIgnore
-  private lazy val kPerActor: Int = k / Constants.defaultNumWorkers
-
-  @JsonIgnore
-  private lazy val kRemainder: Int = k % Constants.defaultNumWorkers
+  private lazy val kPerActor: List[Int] = LimitOpDesc.equallyPartitionGoal(k, Constants.defaultNumWorkers)
```

Comment:
Let's put `equallyPartitionGoal ` into some utility file instead of LimitOpDesc?